proposals/nnnn-addcommand-ui-updates.md

```diff
@@ -0,0 +1,166 @@
+# Main Menu UI Updates
+
+* Proposal: [SDL-NNNN](NNNN-addcommand-ui-updates.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / HMI / RPC]
+
+## Introduction
+This proposal is to update the main menu `AddCommand` and `AddSubMenu` RPCs to allow for the main menu to have the same UI capabilities as `Choice` in `PerformInteraction` popup menus.
+
+## Motivation
+Due to several deficiencies in the `AddCommand` / `AddSubmenu` UI and UX, some developers use `PerformInteraction` popup menus as their primary menu navigation structure. We should try to fix those deficiencies as much as possible. This particular proposal seeks to solve some of the UI differences to enable more information in the main menu cells.
+
+## Proposed solution
+The proposed solution is to add additional parameters to `AddCommand` and `AddSubmenu` to allow main menu UI to display the same information that `PerformInteraction` popup menus do.
+
+### General HMI Documentation Notes
+* It should be an integration guideline that OEMs that implement these UI elements should place them in the same places as the `Choice` UI. In other words, the `AddCommand` / `AddSubmenu` and `Choice` UI should be close to equivalent.
```

Comment:
[SUGGESTION] It should be an integration guideline that OEM's use to implement the UI elements in place of the 'Choice' element on the UI.  In other words, the 'AddCommand' / 'AddSubmenu' and 'Choice' UI should be close to equivalent.

Reply:
I'm confused on what you're trying to say here?